packages/osl-operator-bundle-image/generated/bundle/manifests/logic-operator-rhel8.clusterserviceversion.yaml

```
@@ -0,0 +1,828 @@
+                      - containerPort: 8443
+                        name: https
+                        protocol: TCP
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 128Mi
+                      requests:
+                        cpu: 5m
+                        memory: 64Mi
+                    securityContext:
+                      allowPrivilegeEscalation: false
+                      capabilities:
+                        drop:
+                          - ALL
+                      seccompProfile:
+                        type: RuntimeDefault
+                securityContext:
+                  runAsNonRoot: true
+                serviceAccountName: logic-operator-rhel8-controller-manager
+                terminationGracePeriodSeconds: 10
+                volumes:
+                  - configMap:
+                      name: logic-operator-rhel8-controllers-config
+                    name: controllers-config
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: logic-operator-rhel8-controller-manager
+    strategy: deployment
+  installModes:
+    - supported: false
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - sonataflow
+    - cncf
+    - serverless
+    - serverlessworkflow
+    - workflows
+  links:
+    - name: Documentation
+      url: https://docs.openshift.com/serverless/
+    - name: Source Repository
+      url: https://github.com/kiegroup/kogito-serverless-operator/
+  maintainers:
+    - email: serverless-logic-team@redhat.com
+      name: Red Hat
+  maturity: alpha
+  minKubeVersion: 1.23.0
+  provider:
+    name: Red Hat
+  relatedImages:
+    - image: registry.redhat.io/openshift-serverless-1/logic-rhel8-operator@sha256:ffae74d76431b428341abfe8b8b704323261c224ff9140a2ea2782c0524f222f
+      name: manager
+    - image: gcr.io/kubebuilder/kube-rbac-proxy@sha256:d4883d7c622683b3319b5e6b3a7edfbf2594c18060131a8bf64504805f875522
+      name: kube-rbac-proxy
+  replaces: logic-operator-rhel8.v1.34.0
```

This should be 1.35.0, not sure what part of the automation updates that, but it seems it missed.

```suggestion
  replaces: logic-operator-rhel8.v1.35.0
```